backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -312,6 +312,7 @@ private String checkSlackChannel(CamelProperties camelProperties) {
     @Path("/system/email_subscription")
     @Consumes(APPLICATION_JSON)
     @Produces(APPLICATION_JSON)
+    @Operation(summary = "Create an email subscription endpoint", description = "Adds the email subscription endpoint into the system and specifies the RBAC group that will receive email notifications. Use this endpoint on behavior groups to send emails when an action linked to the behavior group is triggered.")
```

Comment:
I would spell out RBAC - Role Based Access Control.

Reply:
```suggestion
    @Operation(summary = "Create an email subscription endpoint", description = "Adds the email subscription endpoint into the system and specifies the Role Based Access Control group that will receive email notifications. Use this endpoint on behavior groups to send emails when an action linked to the behavior group is triggered.")
```

Reply:
Updated in the file.